DESCRIPTION

```diff
@@ -29,10 +29,10 @@ Imports:
     rversions,
     stats,
     git2r (>= 0.10.1)
-Suggests: 
+Suggests:
     testthat (>= 0.7),
     BiocInstaller,
-    Rcpp (>= 0.10.0),
+    Rcpp (>= 0.11.0),
```

Comment:
YES `git2r` must be >= 0.11.0. Otherwise `cred_env()` is not exported and we are unable to establish push to Github. And if you can't push, you can't set the remote tracking branch.